engineapi/proxy_metrics.go

```diff
@@ -0,0 +1,13 @@
+package engineapi
+
+import (
+	longhorn "github.com/longhorn/longhorn-manager/k8s/pkg/apis/longhorn/v1beta2"
+)
+
+func (p *Proxy) MetricsGet(e *longhorn.Engine) (*Metrics, error) {
+	metrics, err := p.grpcClient.MetricsGet(p.DirectToURL(e))
+	if err != nil {
+		return nil, err
+	}
+	return (*Metrics)(metrics), err
```

Comment:
the metrics returned from the grpc call is a metrics type definied in`github.com/longhorn/longhorn-instance-manager/pkg/client`, but here we define another Metrics type. They can be casted to each other w/o issues? Why not just use the type from instance manager?

Reply:
It's a common pattern in longhorn-manager.

> Why not just use the type from instance manager?

Yeah, using the same type is a idea way. We will refactor them in the future.